source/azure_iot.c

```diff
@@ -12,10 +12,10 @@
 
 #include "azure_iot.h"
 #include "azure_iot_private.h"
-
 #include "azure_iot_result.h"
-#include "azure/az_iot.h"
 
+/* Azure SDK for Embedded C includes */
```

Comment:
I am changing lot of header files in my PR:

https://github.com/Azure/azure-iot-middleware-freertos/pull/201


Also taken care of this ... can we close this so that we have less conflict?